whipper/result/result.py

```diff
@@ -153,7 +153,7 @@ def getLoggers():
     """
     Get all logger plugins with entry point 'whipper.logger'.
 
-    @rtype: dict of C{str} -> C{Logger}
+    :rtype: dict of str -> Logger
```

Comment:
On SourceForge it says:
> C{...}: Source code or a Python identifier.

Example:
```python
def example():
    """
    I{B{Inline markup} may be nested; and
    it may span} multiple lines.

      - I{Italicized text}
      - B{Bold-faced text}
      - C{Source code}
      - M{Math}

    Without the capital letter, matching
    braces are not interpreted as markup:
    C{my_dict={1:2, 3:4}}.
    """
```

In this case I think `class` is the correct one.

Reply:
Ah. That may be then. It does seem like all the cases where it was used here were for Python classes (either stdlib ones or whipper defined ones).